include/stdlib_coroutine.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+/**
+ * @file stdlib_coroutine.hpp
+ * @author Alecto Perez (perez.cs@pm.me)
+ * @brief This file defines a compatibility layer for C++20 coroutines
+ * @date 2021-09-23
+ *
+ * @copyright Copyright (c) 2021
```

Comment:
Please remove the copyright notice, date and something. If you want to know who wrote this code, git blame is your friend.